File: old/Servidor/src/main/java/com/gotravel/model/Ubicacion.java
package com.gotravel.model;

import jakarta.persistence.*;
import lombok.Getter;
import lombok.Setter;

import java.math.BigDecimal;

@Getter
@Setter
@Entity
@Table(name = "ubicacion")
public class Ubicacion extends Localizacion {

    @Column(name = "coordenada_x", nullable = false, precision = 9, scale = 6)
    private BigDecimal coordenadaX;

    @Column(name = "coordenada_y", nullable = false, precision = 9, scale = 6)
    private BigDecimal coordenadaY;

}

File: GoTravelMobile/app/src/main/java/com/gotravel/mobile/data/model/Ubicacion.kt
package com.gotravel.mobile.data.model

import lombok.AllArgsConstructor
import lombok.Getter
import lombok.NoArgsConstructor
import lombok.Setter
import java.math.BigDecimal

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
data class Ubicacion(
    override val id: Int,
    val coordenadaX: BigDecimal,
    val coordenadaY: BigDecimal
) : Localizacion(id)